booting_windows/setup.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -euo pipefail
+
+# Path to a working Windows 2019 image:
+WINDOWS_DISK=$(WORKSPACE)/win2019.raw
```

Comment:
$(WORKSPACE) should be ${WORKSPACE}
... we're trying to name an environment variable, not execute a command called WORKSPACE

Reply:
Fixed. See b28937d6